packages/@ama-sdk/core/README.md

```diff
@@ -42,3 +42,10 @@ A list of API Clients are provided by this package:
 | ApiFetchClient   | @ama-sdk/core                            | Default API Client based on the browser FetchApi                               |
 | ApiBeaconClient  | @ama-sdk/core                            | API Client based on the browser BeaconApi, it is processing synchronous call   |
 | ApiAngularClient | @ama-sdk/core/clients/api-angular-client | API Client using the HttpClient exposed by the `@angular/common` package       |
+
+### Logs
+
+In order to ease the logging in the ama-sdk plugins, it is possible to connect to third-party logging services.
+This can be achieved by adding a `Logger` [implementation](/packages/@ama-sdk/core/src/logger) to the options of an API client (or as a parameter of the plugin for some cases).
```

Comment:
```suggestion
This can be achieved by adding a `Logger` [implementation](/packages/@ama-sdk/core/src/logger/logger.ts) to the options of an API client (or as a parameter of the plugin for some cases).
```